mattermost-community/mattermost-plugin-google-drive
# Adding markdown escape to rendered content

### Summary
Escape markdown

### Ticket
https://mattermost.atlassian.net/browse/MM-61984

### Test cases
1. Create a doc with your Mattermost user
2. Have someone else comment on the doc with some markdown in the comment. For example you can prefix the comment with: > 

**Expected result:** You receive a notification in Mattermost for the comment and the comment markdown is escaped and not rendered as markdown

3. Add a reply to that comment by the same user using markdown again

**Expected result:** You receive a notification in Mattermost for the comment and the comment markdown is escaped and not rendered as markdown
